omero_mapr/mapr_settings.py

```diff
@@ -57,16 +80,17 @@ def config_list_to_dict(config_list):
             "ns": [
                 "openmicroscopy.org/mapr/sirna/supplementary"
             ],
-            "label": ["siRNA supplementary"]
+            "label": "siRNA supplementary"
         }
     },
+
     {
         "menu": "phenotype",
         "config": {
             "default": ["Phenotype"],
             "all": ["Phenotype", "Phenotype Term Accession"],
             "ns": ["openmicroscopy.org/mapr/phenotype"],
```

Comment:
should be good to have the namespace in one location ``openmicroscopy.org/mapr``